demo/nuxt3/CHANGELOG.md

```
@@ -0,0 +1,47 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [3.0.1](https://github.com/tsparticles/vue3/compare/v3.0.0...v3.0.1) (2024-03-06)
+
+**Note:** Version bump only for package nuxt3-particles-demo
+
+# [3.0.0](https://github.com/tsparticles/vue3/compare/v2.12.0...v3.0.0) (2023-12-22)
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)